WooCommerce/Classes/ViewRelated/Products/Variations/Edit Attributes/AttributePickerViewModel.swift

```diff
@@ -0,0 +1,51 @@
+import Foundation
+import Yosemite
+
+final class AttributePickerViewModel {
+
+    private let variationModel: EditableProductVariationModel
+    private var editableAttributes: [ProductVariationAttribute]
+
+    init(variationModel: EditableProductVariationModel) {
+        self.variationModel = variationModel
+        self.editableAttributes = variationModel.productVariation.attributes
+    }
+
+    // MARK: DataSource
+
+    var attributes: [ProductAttribute] {
+        variationModel.allAttributes
+    }
+
+    var isChanged: Bool {
+        variationModel.productVariation.attributes.sorted { $0.id < $1.id } != editableAttributes.sorted { $0.id < $1.id }
+    }
+
+    var resultAttributes: [ProductVariationAttribute] {
+        editableAttributes
+    }
+
+    func selectedOption(for attribute: ProductAttribute) -> ProductVariationAttribute? {
+        return editableAttributes.first(where: { $0.id == attribute.attributeID && $0.name == attribute.name })
+    }
```

Comment:
We tried to keep these public variables documented trough our classes, would you mind adding some to it?
Specially to `isChanged` as it does some calculation!